Source/Bugsnag.h

```diff
@@ -29,10 +29,6 @@
 #import "BugsnagMetadata.h"
 #import "BugsnagPlugin.h"
 
-static NSString *_Nonnull const BugsnagSeverityError = @"error";
-static NSString *_Nonnull const BugsnagSeverityWarning = @"warning";
-static NSString *_Nonnull const BugsnagSeverityInfo = @"info";
```

Comment:
Should we include in Upgrade Guide in case anyone uses them?

Reply:
Added a note to the upgrade guide in case anyone was using these.